edx_proctoring/templates/proctoring/seq_timed_exam_expired.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+<div class="critical-time sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You did not complete the exam in the allotted time
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Since you did not submit your exam before the time allotted expired, your work cannot be
```

Comment:
"You did not submit your answers before the allotted time expired, so your exam is not eligible for credit."